lib/when_committed.rb

```diff
@@ -1,30 +1,42 @@
 require 'when_committed/version'
+require 'active_record'
 
-module WhenCommitted
-  module ActiveRecord
-    def self.included(base)
-      base.after_commit :run_when_committed_callbacks
-      base.after_rollback :clear_when_committed_callbacks
+module WhenCommitted::ActiveRecord
+
+  extend ActiveSupport::Concern
```

Comment:
You're introducing a dependency on activesupport but it's not in the gemfile.  https://github.com/PeopleAdmin/when_committed/blob/master/when_committed.gemspec#L22